capstone-app/src/app/modules/onboarding/content/content.component.scss

```diff
@@ -0,0 +1,26 @@
+h1 {
+    align-items: center;
+    text-align: center;
+}
+
+img {
+    height: 400px;
+}
+
+@media only screen and (max-width: 600px) {
+    mat-panel-title ::ng-deep {
+        font-size: 12px;
+        height: fit-content;
+        margin: 0;
+    }
+
+    mat-panel-description ::ng-deep {
```

Comment:
I think in general we want another selector after ng-deep but I could be wrong

Reply:
+1

Reply:
yes i agree but i don't why that does seem to work for me like I have tried many different combinations of selector after ng-deep or both selector before and selector after. I also tried class name ng-deep selector and vice versa and neither have worked. the only thing that work was just having a selector followed by ng-deep.